packages/plugins/other/visitor-plugin-common/src/base-visitor.ts

```diff
@@ -186,6 +187,10 @@ export interface RawConfig {
    * @ignore
    */
   globalNamespace?: boolean;
+  /**
+   * @ignore
```

Comment:
Why `ignore`? I think we need to document it :)

Reply:
You can define it here, and then you don't need to define it in the config of typed-document-node. This is relevant for all of the plugins that inherits.

Reply:
It's documented [here](https://github.com/dotansimha/graphql-code-generator/blob/c47bcba43904ca6abd1da4209e07011ddd7c08d2/packages/plugins/typescript/typed-document-node/src/config.ts#L26), do you think it is relevant to activate it from the base visitor as well?